index.html

```diff
@@ -234,5 +234,38 @@ <h3>Skills</h3>
         </ul>
       </div>
     </div>
-  </section>  
+  </section>
+  <footer class="form-body">
+    <div class="card-contact">
+      <h2 class="contact-text">Contact me</h2>
```

Comment:
- [ ] Your form looks great, but based on requirements we should stick to Figma template, in the workplace, you may be required to develop a page based on provided design. So let's add more width to form input as provided in Figma: 

Your project  | Figma
------------- | -------------
<img width="374" alt="Screenshot 2022-02-15 at 18 53 53" src="https://user-images.githubusercontent.com/77038610/154087027-a52cdf91-dab8-4b88-952a-03ba2408a1e2.png"> | <img width="390" alt="Screenshot 2022-02-15 at 18 53 04" src="https://user-images.githubusercontent.com/77038610/154086842-a4500404-71e1-49a6-8252-8cbd527a186f.png">

Reply:
Thanks for the review. I have done the changes